.github/workflows/benchmarks-execute.yml

```diff
@@ -66,12 +66,12 @@ jobs:
 
       - name: Build benchmarks
         working-directory: benchmarks/execute
-        run: cargo codspeed build --profile maxperf
+        run: cargo $TOOLCHAIN codspeed build --profile maxperf --features tco
```

Comment:
@shuklaayush i'm not sure what the right syntax is here

Reply:
i think you only need to add `--features tco` to build, not run